docs/insomnia/get-started/mock-server/allowlist.md

```diff
@@ -3,60 +3,29 @@ layout: article-detail
 title:  Allowlist for Insomnia
 ---
 
-<!-- for docs rework: revise this page like we normally would -->
+For enterprise users utilizing Insomnia, it's crucial to ensure that specific domains are allowlisted in your network. This step is essential to guarantee that all features of Insomnia, including updates, synchronization, documentation, and necessary backend services work without any hindrance in environments with restricted internet access.
```

Comment:
@filfreire I didn't edit this paragraph yet because I wanted to double check with you first: this doc applies to any users who have restricted internet access, right? Not just enterprise customers?